frontend/TestRun/ResultsTab.svelte

```diff
@@ -177,14 +178,20 @@
                 fltrs.get(level).add(part.trim());
             });
         });
-        filters = Array.from(fltrs.entries()).sort((a, b) => a[0] - b[0]).map(entry => ({
-            level: entry[0],
-            items: Array.from(entry[1])
-        }))
+
+        filters = Array.from(fltrs.entries())
+            .sort((a, b) => a[0] - b[0])
```

Comment:
Small tip here, you can use `([left], [right]) => left - right` form to destructure array into named values